Multiple inhand sprites fix

kingofkosmos opened the pull request:
Fixes One-point bluespace-gravitational manipulator's inhand.
Adds inhand sprites for the Chem Sprayer & DRAGnet. (Thanks to @FuryMcFlurry for the sprites!)

Fixes #29096 
Fixes #29254 
Fixes #28043 

:cl:
imageadd: Inhands for One-point bluespace-gravitational manipulator, DRAGnet and Chem Sprayer.
/:cl:


~~Also adds placeholder-inhands for all the severed limbs - a gray block, same as a pipe, but it's better than nothing. If someone wants to make better inhands for the limbs, I'll add them here.~~
~~Still waiting for the inhand sprites for:~~
* ~~Chem Sprayer~~
* ~~DRAGnet~~
~~Will add once someone makes them (already requested on the forums).~~

RemieRichards commented on code/modules/surgery/bodyparts/bodyparts.dm:
```diff
@@ -2,6 +2,7 @@
 /obj/item/bodypart
 	name = "limb"
 	desc = "why is it detached..."
+	item_state = "buildpipe"
```
No.
Flesh limbs should not show up as pipes, that's dumb.

DoctoreMad commented:
>Also adds placeholder-inhands for all the severed limbs - a gray block, same as a pipe

Well that's not really necessary. If we get proper limb sprites we can use them, if we don't have them, its not like its some sort of lethal item or anything so why do you need to change it?

kingofkosmos commented:
>Well that's not really necessary. If we get proper limb sprites we can use them, if we don't have them, its not like its some sort of lethal item or anything so why do you need to change it?

I thought that anything in hand would add to the "feel" of the game. If people disagree I can remove it from the PR and wait for proper inhands if there will be any.

optimumtact commented:
it's really quite remarkable that dismembered limbs got in without an inhand

RemieRichards commented:
Not really, at the time we had poor support for colour inhands (read: none), no support for overlays on inhands, etc.

Not to mention they don't DO anything, they're not weapons and don't specifically -need- an inhand, certainly not enough to justify a grey pipe.

AnturK commented:
Yeah, just skip the inhands for now , we'll just wait bit longer.

Cyberboss reviewed (changes requested):
Do the thing remie wants

FuryMcFlurry commented:
I've contributed dragnet and chemsprayer inhands. Awaiting change.

Tacolizard commented:
pics or GTFO

Cyberboss commented:
@kingofkosmos

FuryMcFlurry commented:
@Tacolizard 
![image](https://user-images.githubusercontent.com/8770071/28726194-90e01480-73c0-11e7-847f-934590091b2a.png)

guess which one is which

kingofkosmos committed:
Adds inhand sprites for DRAGnet and Chemical Sprayer, fixes One-point manipulator's in hand.

kingofkosmos commented:
Original post edited. Added Fury's DRAGnet and Chem Sprayer inhands.

Removed severed limbs stuff. Unless someone wants to make them, I'll add them to this PR. Btw should there be an inhand for every limb? Or just for heads and a some sort of blob-lookalike for all the other limbs + chest?
